lib/core/theming/colors.dart

```diff
@@ -1,5 +1,7 @@
 import 'dart:ui';
 
 class AppColors {
-  static const Color mainColor = Color(0xff123123);
-}
\ No newline at end of file
+  static const Color primairyColor = Color(0xffFF1768);
```

Comment:
think of a better name than "primairyColor", something like "mainBlue" ... "lightBlue" .. etc.